en/advanced/gimbal_control.md

```diff
@@ -1,82 +1,72 @@
+
+You should set `MNT_MODE_IN` to one of: `RC (1)`, `MAVlink gimbal protocol v2 (4)` or `Auto (0)` (the other options are deprecated).
+If you select `Auto (0)`, the gimbal will automatically select either RC or or MAVLink input based on the latest input.
+Note that the auto-switch from MAVLink to RC requires a large stick motion!
+
+The output is set using the [MNT_MODE_OUT](../advanced_config/parameter_reference.md#MNT_MODE_OUT) parameter.
+By default the output is set to a PXM port (`AUX (0)`).
+If the [MAVLink Gimbal Protocol v2](https://mavlink.io/en/services/gimbal_v2.html) is supported by your gimbal, you should instead select `MAVLink gimbal protocol v2 (2)`.
+
+
+The full list of parameters for setting up the mount driver can be found in [Parameter Reference > Mount](../advanced_config/parameter_reference.md#mount).
+The relevant settings for a number of common gimbal configurations are described below.
 
 ## MAVLink Gimbal (MNT_MODE_OUT=MAVLINK)
 
-To enable a MAVLink gimbal, first set parameter [MNT_MODE_IN](../advanced_config/parameter_reference.md#MNT_MODE_IN) to `MAVLINK_DO_MOUNT` and [MNT_MODE_OUT](../advanced_config/parameter_reference.md#MNT_MODE_OUT) to `MAVLINK`.
+To enable a MAVLink gimbal, first set parameter [MNT_MODE_IN](../advanced_config/parameter_reference.md#MNT_MODE_IN) to `MAVlink gimbal protocol v2` and [MNT_MODE_OUT](../advanced_config/parameter_reference.md#MNT_MODE_OUT) to `MAVLink gimbal protocol v2`. 
 
 The gimbal can be connected to *any free serial port* using the instructions in [MAVLink Peripherals (GCS/OSD/Companion)](../peripherals/mavlink_peripherals.md) (also see [Serial Port Configuration](../peripherals/serial_configuration.md#serial-port-configuration)).
-
-A common configuration is to have a serial connection to the gimbal from the Flight Controller TELEM2 port (assuming TELEM2 is free).
-For this configuration you would set:
+For example, if the `TELEM2` port on the flight controller is unused you can connect it to the gimbal and set the following PX4 parameters:
 - [MAV_1_CONFIG](../advanced_config/parameter_reference.md#MAV_1_CONFIG) to **TELEM2** (if `MAV_1_CONFIG` is already used for a companion computer (say), use `MAV_2_CONFIG`).
 - [MAV_1_MODE](../advanced_config/parameter_reference.md#MAV_1_MODE) to **NORMAL**
 - [SER_TEL2_BAUD](../advanced_config/parameter_reference.md#SER_TEL2_BAUD) to manufacturer recommended baud rate.
 
-This will enable the user to command the gimbal using [MAV_CMD_DO_MOUNT_CONTROL](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_MOUNT_CONTROL) and [MAV_CMD_DO_MOUNT_CONFIGURE](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_MOUNT_CONFIGURE).
+With this setup, ground stations can send [MAVLink Gimbal Commands](https://mavlink.io/en/services/gimbal_v2.html#gimbal-manager-messages) to PX4, such as `MAV_CMD_DO_SET_ROI_LOCATION` to track region of interest and `MAV_CMD_DO_GIMBAL_MANAGER_PITCHYAW` to control the gimbal attitude, and so on.
 
+:::note
```

Comment:
@julianoes I'm going to merge this as "better than it was" and we can iterate. I think this is probably close to correct, but perhaps not "complete"

However can you re-review this for sanity?

For example, this note states my assumption that this setup makes PX4 a gimbal manager. That might not be so.

A few more questions:

1. Are there any mavlink gimbal protocol v2 we know might work?
2. If they use an old protocol gimbal, can you still drive it using the new protocol messages - e.g. such as setting the ROI. Similarly, can I drive a PWM gimbal with all the instructions in the gimbal manager docs. The docs say that the gimbal manager will translate the input to the format expected but I suspect it might be more nuanced than that? Or to put it another way, does the PX4 gimbal manager do everything you might expect to convert all gimbal manager commands to roll, pitch, yaw ?
3. If not, what isn't supported?
4. Given the gimbal manager does all those things (say) why would we recommend a mavlink gimbal over using PX4 with a PWM connected gimbal?
5. What should someone do if there is a mavlink gimbal on the system that supports gimbal manager? Do we need to document that?

Reply:
1. Gremsy
2. Yes, PX4 translates between v2 and v1, or v2 and PWM. So it should do it all. If you find something that doesn't work right, it's a bug.
3. Don't know.
4. If you want to fiddle with PWM, feel free. If something should just work then v2 would be recommended
5. You mean a gimbal that implements the manager? In that case don't enable gimbal control in PX4. In that case commands and mission items would be sent to the gimbal manager.
6. QGC is not really maintained or updated so :shrug:.